src/TitleDetails/TitleInformation/TitleInformation.js

```diff
@@ -41,6 +42,15 @@ const TitleInformation = ({
     )
     : title;
 
+  const poLineNumberValue = (
+    <Link
+      data-testid="titlePOLineLink"
+      to={`/orders/lines/view/${poLineId}`}
```

Comment:
IMHO, it's also should have filter query so list of PO Lines is filtered by particular PO Line number, so this specific po line is displayed in the list

Reply:
Thanks, good point.